MastodonSDK/Sources/MastodonCore/Service/InstanceService.swift

```diff
@@ -24,7 +24,8 @@ public final class InstanceService {
     weak var authenticationService: AuthenticationService?
     
     // output
-    
+    public let updateActiveUserAccountPublisher = PassthroughSubject<Void, Never>()
```

Comment:
I'm actually not entirely sure if the `InstanceService` is the best place to put this subject, but I thought it's most appropriate as the active user is tied to the instance.

Reply:
Maybe `AuthenticationService`? 

The active user info is isolated to `AuthContext`. So that's fine to bind to any service that could access or handle `AuthContext`.

Reply:
Yes, maybe that's better. 👍  I'll move it there and raise another PR as I've merged this one already.